tests/sqlparser_hive.rs

```diff
@@ -534,6 +534,20 @@ fn parse_use() {
     );
 }
 
+#[test]
+fn test_show() {
+    hive_and_generic().verified_stmt("SHOW DATABASES");
+    hive_and_generic().verified_stmt("SHOW DATABASES LIKE '%abc'");
+    hive_and_generic().verified_stmt("SHOW SCHEMAS");
+    hive_and_generic().verified_stmt("SHOW SCHEMAS LIKE '%abc'");
+    hive_and_generic().verified_stmt("SHOW TABLES");
+    hive_and_generic().verified_stmt("SHOW TABLES IN db1");
+    hive_and_generic().verified_stmt("SHOW TABLES IN db1 'abc'");
```

Comment:
for tables/views in addition to `IN` can we also include testcases for `FROM`?

Reply:
Sure, according to the hive docs, it's only for views though.